libr/debug/p/debug_gdb.c

```diff
@@ -311,6 +312,21 @@ static int r_debug_gdb_reg_write(RDebug *dbg, int type, const ut8 *buf, int size
 		}
 		ut64 val = r_reg_get_value (dbg->reg, current);
 		int bytes = bits / 8;
+		if (r_config_get_i(((RCore*)dbg->corebind.core)->config, "cfg.bigendian")) {
```

Comment:
Add space before (.. and i dont like the idea of calling config from here. It will break thendependency hirearchy i think. Can we have a gdbwrap api to set the endian and call it from outside?

Reply:
It did require to link against r_config, but I'm not sure it's that bad - it's already linking against many core libraries.

What's gdbwrap?